src/reanimated2/globals.d.ts

```diff
@@ -1,143 +1,96 @@
+/* eslint-disable no-var, @typescript-eslint/no-explicit-any */
```

Comment:
Why do we need to ignore these rules? Can we ignore only specific lines?

Reply:
We have vars everywhere so ignoring every single line would be... pretty bad. I moved `no-explicit-any` to specific lines though.